src/test/regress/expected/grant_on_database_propagation_from_non_maindb.out

```diff
@@ -0,0 +1,470 @@
+\c regression - - :master_port
+select check_database_privileges('myuser7','test_2pc_db',ARRAY['CREATE', 'CONNECT', 'TEMP', 'TEMPORARY']);
+ check_database_privileges
+---------------------------------------------------------------------
+ (CREATE,t)
+ (CREATE,t)
+ (CREATE,t)
+ (CONNECT,t)
+ (CONNECT,t)
+ (CONNECT,t)
+ (TEMP,t)
+ (TEMP,t)
+ (TEMP,t)
+ (TEMPORARY,t)
+ (TEMPORARY,t)
+ (TEMPORARY,t)
+(12 rows)
+
+\c test_2pc_db - - :master_port
+reset role;
+revoke  CREATE,CONNECT,TEMP,TEMPORARY on database test_2pc_db from myuser7;
+revoke CREATE,CONNECT,TEMP,TEMPORARY on database test_2pc_db from myuser_1;
+\c regression - - :master_port
+drop user myuser_1;
+drop user myuser7;
+---------------------------------------------------------------------
+-- test CREATE,CONNECT,TEMP,TEMPORARY privileges one by one on database multi database
+-- and multi user
+\c regression - - :master_port
+create user myuser8;
+create user myuser_2;
+set citus.enable_create_database_propagation to on;
+create database test_db;
+revoke connect,temp,temporary  on database test_db from public;
+\c test_2pc_db - - :master_port
+grant CREATE,CONNECT,TEMP,TEMPORARY on database test_2pc_db,test_db to myuser8,myuser_2;
+\c regression - - :master_port
+select check_database_privileges('myuser8','test_2pc_db',ARRAY['CREATE', 'CONNECT', 'TEMP', 'TEMPORARY']);
+ check_database_privileges
+---------------------------------------------------------------------
+ (CREATE,t)
+ (CREATE,t)
+ (CREATE,t)
+ (CONNECT,t)
+ (CONNECT,t)
+ (CONNECT,t)
+ (TEMP,t)
+ (TEMP,t)
+ (TEMP,t)
+ (TEMPORARY,t)
+ (TEMPORARY,t)
+ (TEMPORARY,t)
+(12 rows)
+
+select check_database_privileges('myuser8','test_db',ARRAY['CREATE', 'CONNECT', 'TEMP', 'TEMPORARY']);
+ check_database_privileges
+---------------------------------------------------------------------
+ (CREATE,t)
+ (CREATE,t)
+ (CREATE,t)
+ (CONNECT,t)
+ (CONNECT,t)
+ (CONNECT,t)
+ (TEMP,t)
+ (TEMP,t)
+ (TEMP,t)
+ (TEMPORARY,t)
+ (TEMPORARY,t)
+ (TEMPORARY,t)
+(12 rows)
+
+select check_database_privileges('myuser_2','test_2pc_db',ARRAY['CREATE', 'CONNECT', 'TEMP', 'TEMPORARY']);
+ check_database_privileges
+---------------------------------------------------------------------
+ (CREATE,t)
+ (CREATE,t)
+ (CREATE,t)
+ (CONNECT,t)
+ (CONNECT,t)
+ (CONNECT,t)
+ (TEMP,t)
+ (TEMP,t)
+ (TEMP,t)
+ (TEMPORARY,t)
+ (TEMPORARY,t)
+ (TEMPORARY,t)
+(12 rows)
+
+select check_database_privileges('myuser_2','test_db',ARRAY['CREATE', 'CONNECT', 'TEMP', 'TEMPORARY']);
+ check_database_privileges
+---------------------------------------------------------------------
+ (CREATE,t)
+ (CREATE,t)
+ (CREATE,t)
+ (CONNECT,t)
+ (CONNECT,t)
+ (CONNECT,t)
+ (TEMP,t)
+ (TEMP,t)
+ (TEMP,t)
+ (TEMPORARY,t)
+ (TEMPORARY,t)
+ (TEMPORARY,t)
+(12 rows)
+
+\c test_2pc_db - - :master_port
+RESET ROLE;
+--below test should fail and should throw an error
+revoke  CREATE,CONNECT,TEMP,TEMPORARY on database test_2pc_db,test_db from myuser8 ;
```

Comment:
Comment says that this should fail. Is it an outdated comment?

Reply:
You're right outdated comment